README.md

```diff
@@ -44,24 +44,38 @@ Fly is extremely extensible, so _anything_ can be a task. Our core system will a
 <h2>Table of Contents</h2>
 <details>
 <summary>Table of Contents</summary>
-- [Features](#features)
```

Comment:
@jbucaran Yep, seems like GitHub [updated](https://github.com/blog/2333-a-formal-spec-for-github-flavored-markdown) its `markdown` processing. So need to check spec and update all repos.